src/pixbuf.rs

```diff
@@ -266,12 +280,23 @@ impl Pixbuf {
     }
 
     pub fn put_pixel(&self, x: i32, y: i32, red: u8, green: u8, blue: u8, alpha: u8) {
```

Comment:
Also: why not taking `u32` instead of `i32`? And since it's not an `unsafe` function, I'm a lot less confident with the assert here...

Reply:
Changed it to u32.  I've used an assert as an argument check, so programs will be fixed early.

Reply:
:+1: